native/packages/react-native-bpk-component-badge/README.md

```diff
@@ -34,18 +35,36 @@ export default class App extends Component {
         <BpkBadge message="Badge" type={BADGE_TYPES.inverse} />
         <BpkBadge message="Badge" type={BADGE_TYPES.light} />
         <BpkBadge message="Badge" type={BADGE_TYPES.outline} />
-        <BpkBadge message="Badge" type={BADGE_TYPES.success} />
+        <BpkBadge
+          message="Badge"
+          type={BADGE_TYPES.success}
+          accessoryView={
+            <BpkBadgeIcons
+              icons={[<BpkIcon icon={icons.flight} />, <BpkIcon icon={icons.hotels} />]}
+              separator="+"
+            />
+          }
+        />
         <BpkBadge message="Badge" docked={BADGE_DOCKED_TYPES.start} type={BADGE_TYPES.warning} />
         <BpkBadge message="Badge" docked={BADGE_DOCKED_TYPES.end} type={BADGE_TYPES.destructive} />
       </View >
     );
   }
 }
 ```
+
 ## Props
 
-| Property              | PropType                                                                 | Required | Default Value |
+| Property              | PropType                                                                  | Required | Default Value |
 | --------------------- | --------------------------------------------------------------------------| -------- | ------------- |
 | message               | string                                                                    | true     | -             |
+| accessoryView         | node                                                                      | false    | null          |
 | docked                | oneOf('start', 'end')                                                     | false    | null          |
 | type                  | oneOf('success', 'warning', 'destructive', 'light', 'inverse', 'outline') | false    | warning       |
+
+
+## Accessory view
```

Comment:
```suggestion
## `accessoryView`
 The accessory view allows for icons to be placed in front of the text inside the Badge component in conjunction with `BpkBadgeIcons`.

## `BpkBadgeIcons`

Renders icons suitable to be placed inside a Badge.

### Props

<props table here for BpkBadgeIcons here>
```

I based this off of https://backpack.github.io/components/navigation-bar?platform=native